### DISCUSSION TITLE
Ignore dependency in some packages

### DISCUSSION THREAD

**Discussion by dyladan** [renovatebot/renovate#10814] [Request Help]
Hello. I am a maintainer of OpenTelemetry JS. We currently have a monorepo architecture, and as a part of our project we have 3 packages which depend on older versions of `@types/node` as backward compatibility checks to ensure our library will still compile if users use those older node versions. Renovate keeps trying to update them to the latest `@types/node` packages. We have tried several renovate configurations but have had no luck at all in getting renovate to ignore them.

I want renovate to not update `@types/node` in any package in the `backwards-compatibility` directory. Can someone please help me with this?

**Comment by viceice** [comment]
try this package rule
```json
{
  "matchPaths": ["backwards-compatibility"],
  "matchPackageNames": ["@types/node"],
  "enabled": false
}
```
